release-team/role-handbooks/bug-triage/README.md

```diff
@@ -1,195 +1,223 @@
+You can [refer to this guide](https://github.com/kubernetes/community/blob/master/contributors/guide/issue-triage.md#define-priority).
+
+
+### Sample Searches [Before Freeze]
+* Open v1.14 Issues
+   * `is:open is:issue milestone:v1.14 -label:"kind/feature" -label:"kind/failing-test"` [milestone open issues, excluding kind/feature and kind/failing-test](https://github.com/kubernetes/kubernetes/issues?utf8=%E2%9C%93&q=is%3Aopen+is%3Aissue+milestone%3Av1.14+-label%3A%22kind%2Ffeature%22+-label%3A%22kind%2Ffailing-test%22+)
+   * `is:open is:issue milestone:v1.14` [milestone open issues, no exclusions](https://github.com/kubernetes/kubernetes/issues?utf8=%E2%9C%93&q=is%3Aopen+is%3Aissue+milestone%3Av1.14)
+* Open v1.14 PRs
+   * `is:open is:pr milestone:v1.14 -label:"kind/feature" -label:"kind/failing-test"` [milestone open PRs, excluding kind/feature and kind/failing-test](https://github.com/kubernetes/kubernetes/pulls?utf8=%E2%9C%93&q=is%3Aopen+is%3Apr+milestone%3Av1.14+-label%3A%22kind%2Ffeature%22+-label%3A%22kind%2Ffailing-test%22+)
+  * `is:open is:pr milestone:v1.14"` [milestone open PRs, no exclusions](https://github.com/kubernetes/kubernetes/pulls?utf8=%E2%9C%93&q=is%3Aopen+is%3Apr+milestone%3Av1.14)
+
+
+### Reports [Before Freeze]
+
+At this point, release meetings start occurring ~3 times per week and a report is expected in each of them.
+
+You should soon start maintaining an overall list of items/areas/issues which seem to be delaying, at risk of not making it in time, or in need of attention by the Release Lead or other roles, and present it in the meetings.
 
 The Release Lead will include a template for this information in the Monday/Wednesday/Friday release team meeting notes.
 
+Code Freeze hasn't really started yet at this point, and it depends on the overall status and volume of open issues/PRs on how many to report. It's good to have some numbers of total issues/PRs available to see general trends over time, like increased or lowered volumes.
+
+### Issue Categorization
+
+Issues can be categorized with the following 'status codes', Green/Yellow/Red.
+
+* green: issues/PRs where the resolution is known and will be resolved soon
+* yellow: issues/PRs which are in progress but are not as easy and need to be watched
+* red: issues/PRs that are heavily release-impacting, and/or with a likelihood of not making it in time.
+
+In each meeting, it is good to give an overall status of Bug Triage: Green, Yellow or Red, generally based on the amount of issues from each category. Examples are:
+
+- status can be Red if there are multiple release-impacting issues that are in danger of not being fixed in the release.
+- status can be Yellow if there are multiple issues open but they are being worked on, which is the usual state of the world.
+- status can be Green if there are no Red issues and workpaces seem normal.
+
+It is useful to keep Red issues on a separate section and report them during the meetings.
```

Comment:
I love this entire section - very specific and helpful!
This is a perfect example of picking up on "oral tradition" and making it legit. I wasn't even aware this was missing before. ❤️

Reply:
It was there in some way but stricter and duplicated 4 times throughout the 'reports' sections, with slightly different wording.. made it one and unified for clarity